python/llm/test/benchmark/csv_to_html.py

```diff
@@ -96,9 +136,15 @@ def main():
         latest_csv.insert(loc=5,column='last2',value=last2)
         latest_csv.insert(loc=6,column='diff2(%)',value=diff2)
 
-        subset=['diff1(%)','diff2(%)']
+        latest_csv.insert(loc=7,column='best last1',value=best_last1)
+        latest_csv.insert(loc=8,column='best diff1(%)',value=best_diff1)
+        latest_csv.insert(loc=9,column='best last2',value=best_last2)
+        latest_csv.insert(loc=10,column='best diff2(%)',value=best_diff2)
+
+        subset=['diff1(%)','diff2(%)','best diff1(%)','best diff2(%)']
         columns={'1st token avg latency (ms)': '{:.2f}', '2+ avg latency (ms/token)': '{:.2f}', 'last1': '{:.2f}', 'diff1(%)': '{:.2f}',
-                'last2': '{:.2f}', 'diff2(%)': '{:.2f}', 'encoder time (ms)': '{:.2f}', 'peak mem (GB)': '{:.2f}'}
+                'last2': '{:.2f}', 'diff2(%)': '{:.2f}', 'encoder time (ms)': '{:.2f}', 'peak mem (GB)': '{:.2f}',
+                'best last1': '{:.2f}', 'best diff1(%)': '{:.2f}', 'best last2': '{:.2f}', 'best diff2(%)': '{:.2f}'}
```

Comment:
Let's change the name to "best 1" and "best 2"; best last 1 and best last 2 is strange :)

Reply:
sure